src/parser/code.rs

```diff
@@ -88,6 +85,10 @@ impl CodeParser {
                         Some(s.to_string())
                     }
                 });
+                let index_str = parts.next().unwrap_or("0");
```

Comment:
Why the `unwrap_or` call instead of failing?

Reply:
At this point, I was not really sure how it would look like in the end, and didn't revisit it. Thanks!